test/cell_multi_cell_refactor/multi_cell.py

```diff
@@ -67,71 +64,10 @@ def test_ln_positioning_and_page_breaking_for_multicell(self):
         doc.cell(w=72 * 2, h=line_height, border=1, ln=2, txt="Lorem ipsum")
         doc.cell(w=72 * 2, h=line_height, border=1, ln=2, txt="Lorem ipsum")
 
-        outfile = relative_path_to("output.pdf")
-        doc.output(outfile)
-        known_good_hash = "a3510b31ecf981b759e5ef9e7f8a47d4"
-        self.assertEqual(calculate_hash_of_file(outfile), known_good_hash)
-        os.unlink(outfile)
-
-
-## Code used to create test
```

Comment:
same as above

Reply:
Same reply :) Do you want me to restore those?